src/server.ts

```diff
@@ -0,0 +1,109 @@
+import 'reflect-metadata';
+import { getRepository } from 'typeorm';
+import { User } from './entities/user';
+import {configServer } from './config';
+import { hashPassword } from './hash-password';
+import {INVALID_ID, INVALID_CREDENTIALS, EMAIL_NOT_FOUND} from './errors';
+
+var jwt = require('jsonwebtoken');
+
+const APP_SECRET = 'PASSWORLD'; // signature
+const TEN_MINUTES = 600;
+const ONE_WEEK = 300000;
+
+const { GraphQLServer } = require('graphql-yoga')
+
+export const startServer = async () => {
+
+  const typeDefs = `
+  input LoginInputType {
+    email: String!
+    password: String!
+    rememberMe: Boolean
+  }
+  type UserType {
+    id: ID!
+    name: String!
+    email: String!
+    birthDate: String!
+    cpf: String!
+  }
+  type LoginType {
+    user: UserType!
+    token: String!
+  }
+  type Query {
+    user(id: ID!): UserType!
+  }
+  type Mutation {
+    login(data:LoginInputType!): LoginType!
+  }
+  `
+  
+  class LoginType {
+    user: User;
+    token: string;
+  
+    constructor( user: User ,token: string ) {
+      this.user = user;
+      this.token = token;
+    }
+  }
+  
+  const resolvers = {
+  
+    Query: {
+      
+      async user (_, { id }){
+        const user: User = await getRepository(User).findOne(id);
+  
+        if (user == undefined) {
+          throw new Error(INVALID_ID);
+        }
+        
+        return user;
+      },
+    },
+    
+    Mutation: {
+  
+      login: async (_, { data } ) => {
+    
+        const user: User = await getRepository(User).findOne({
```

Comment:
Você já deve imaginar que conforme você for adicionando queries/mutations no seu server, esse arquivo vai crescer desenfreadamente, certo? Sugiro você ir pensando numa estratégia para organizar (que pode ser aplicada nos próximos PRs).